dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaJobsTask.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.cloudera.manager;
+
+import com.google.common.io.ByteSink;
+import com.google.edwmigration.dumper.application.dumper.task.TaskRunContext;
+import java.util.List;
+import javax.annotation.Nonnull;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.yarn.api.records.ApplicationReport;
+import org.apache.hadoop.yarn.client.api.YarnClient;
+import org.apache.hadoop.yarn.conf.YarnConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** The task dump job list from YARN and Cloudera Management API */
+public class ClouderaJobsTask extends AbstractClouderaManagerTask {
+  private static final Logger LOG = LoggerFactory.getLogger(ClouderaJobsTask.class);
+
+  public ClouderaJobsTask() {
+    super("jobs.jsonl");
+  }
+
+  @Override
+  protected void doRun(
+      TaskRunContext context, @Nonnull ByteSink sink, @Nonnull ClouderaManagerHandle handle)
+      throws Exception {
+    Configuration conf = new YarnConfiguration();
+    conf.addResource(
+        new org.apache.hadoop.fs.Path(
+            "/opt/cloudera/parcels/CDH/lib/hadoop/etc/hadoop/core-site.xml"));
+    conf.addResource(
+        new org.apache.hadoop.fs.Path(
+            "/opt/cloudera/parcels/CDH/lib/hadoop/etc/hadoop/hdfs-site.xml"));
+    conf.addResource(
+        new org.apache.hadoop.fs.Path(
+            "/opt/cloudera/parcels/CDH/lib/hadoop/etc/hadoop/yarn-site.xml"));
+
+    YarnClient yarnClient = YarnClient.createYarnClient();
+    yarnClient.init(conf);
+    LOG.warn(conf.get("hadoop.security.authentication"));
+    yarnClient.start();
+
+    List<ApplicationReport> applicationReports = yarnClient.getApplications();
+    applicationReports.forEach(
+        appReport ->
+            System.out.printf(
```

Comment:
Please use logs, not `System.out`.  
https://stackoverflow.com/questions/8601831/do-not-use-system-out-println-in-server-side-code

Reply:
Got it, thanks!